LeetCodeAssistant/LeetCodeProblemRepository.swift

```diff
@@ -103,6 +103,37 @@ class LeetCodeProblemRepository {
             }.resume()
         }
     }
+
+func getProblemDetail(titleSlug: String, completion: @escaping (String) -> Void) {
```

Comment:
As matching the identifier with model's one, `titleSlug` is supposed to be `id`.